.circleci/config.yml

```diff
@@ -44,6 +44,11 @@ commands:
             CIRCLE_NODE_INDEX: << parameters.index >>
             CIRCLE_NODE_TOTAL: << parameters.total >>
           command: ./.circleci/test-package.sh fxa-content-server
+      - run:
+          name: Clean up old Stripe customers from testing
+          command: |
+            cd packages/fxa-auth-server && \
+            yarn run clean-up-old-ci-stripe-customers
```

Comment:
should this have a `when: always`? https://circleci.com/docs/2.0/configuration-reference/#the-when-attribute

Reply:
Eh, probably not. It only deletes customers over a day old, so as long as *some* CI run executes this at some point, it'll catch things